libnavigator/src/test/java/com/mapbox/navigation/navigator/NavigatorMapperTest.kt

```diff
@@ -78,19 +84,6 @@ class NavigatorMapperTest {
         assertEquals(RouteAlertType.TunnelEntrance, result.routeAlerts[0].alertType)
     }
 
-    @Test
-    fun `unsupported alert types are ignored in the route init info`() {
```

Comment:
Why we removed this test? 🤔

Reply:
because only one unsupported type was Incidents, and from now it's supporting